graylog2-web-interface/src/components/search/SurroundingSearchButton.test.jsx

```diff
@@ -5,6 +5,7 @@ import { cleanup, fireEvent, render } from 'wrappedTestingLibrary';
 import DrilldownContext from 'views/components/contexts/DrilldownContext';
 import SurroundingSearchButton from './SurroundingSearchButton';
 import type { SearchesConfig } from './SearchConfig';
+import { asElement } from '../../../test/wrappedTestingLibrary';
```

Comment:
This import could be shortened to
```suggestion
import { asElement } from 'wrappedTestingLibrary';
```